lib/elixir_analyzer/constants.ex

```diff
@@ -71,6 +71,9 @@ defmodule ElixirAnalyzer.Constants do
       "elixir.high-score.use_default_argument_with_module_attribute",
     high_score_use_map_update: "elixir.high-score.use_map_update",
 
+    # High School Sweetheart Comments
+    high_school_sweetheart_function_reuse: "elixir.high_school_sweetheart.function_reuse",
```

Comment:
Maybe we should split this comment into 3, because they are about different functions.

Reply:
I did write based on this comment about function reuse, maybe we can maintain this one, what do you think?

https://github.com/exercism/elixir-analyzer/blob/ca5fd50ff10ba572186690473d13b2befd9ccc41/lib/elixir_analyzer/constants.ex#L41